docs/product/issues/issue-details/index.mdx

```diff
@@ -102,6 +102,18 @@ If you’ve enabled [Session Replay](/product/explore/session-replay/), you’ll
 
 ![Session Replay](./img/issue-replay.png)
 
+## Feature Flags
+There are two types of integrations you can configure to integrate feature flag data into your error events.
+
+The first integration type is **flag evaluation tracking** within the Sentry SDK. To enable it, update the Sentry SDK's `init` method to include one of our supported feature flag integrations. Support varies by language and by provider. Doing so will give you access to the feature flag table, which shows any flags that were evaluated after the Sentry SDK's initialization and before the error event occurred. You will see rows of (flag, evaluation-result) pairs. Supported integrations include:
+
+* LaunchDarkly ([Python](/platforms/python/integrations/launchdarkly/) and [JavaScript](/platforms/javascript/integrations/launchdarkly/))
+* OpenFeature ([Python](/platforms/python/integrations/openfeature/) and [JavaScript](/platforms/javascript/integrations/openfeature/))
+
+The second integration type is **feature flag change tracking**. This will enable a timeline of feature flag definition changes on the event and user volume chart. These annotations can help you diagnose when a feature flag change is related to a new issue. To enable this type of integration, please visit the documentation page of one of our supported providers:
```

Comment:
we should list the providers and the links here!

Reply:
we might even want to add sentry docs on how to do it too

Reply:
or at the very least, we should link to a spot where we have a full list of providers (an index page, when we create one, would make sense)

Reply:
PR is out for the index page: https://github.com/getsentry/sentry-docs/pull/11969

can link to `/product/explore/feature-flags/#set-up-your-integration-specific-webhook` for the full list of providers

Reply:
or `/product/explore/feature-flags/#setup` for the full setup info!